### PR TITLE
The Ultimate Electric Boogaloo Edition Of Syndicate Agents Part 2 Episode 2.5 The Rise Of The Spurdo Sparde

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
add: Added the autosurgeon to the syndicate uplink. A vastly improved version of what the Chief Medical Officer starts with, it allows you to instantly implant yourself with any implants that you load in it, and it can be re-used indefinitely.
del: Removed the Sports Bundle...
tweak: ...and separated it into its 4 individual possible bundles! You can now choose between the Hockey Set, the Football Set, the Bowling Set and the Wrestling Set, with their own prices based on how good they are! They can be found in the "Loud and Aggressive" section of your uplink.
balance: The Syndicate has recognized the utility of no-slip shoes, and have removed the player count restriction on purchasing them. Nuclear Operatives may now also buy the chameleon kit independently of the player count.
balance: A multitude amount of traitor items had their prices slightly reduced.
tweak: Did you miss the old days when you could rush and suicide-bomb the shuttle as a traitor by blowing all of your TC on one really large bomb implant? Well, you can now do that again! Traitors can now purchase microbomb and macrobomb implants.
/:cl:

I just hope this is actually modularized. Traitors are currently sort of dull, and a lot of them refuse to try gimmicks due to the fear that they will throw away their round. The mulligan's effect can be achieved with a trip to Genetics, never pick the Sports Bundle for the fear that they'll waste all of their TC on the bowling set, the small things that deter people from picking the unique, specialized traitor items in favor of the practical emag or energy sword. Due to their under-usage, the prices of several items have been reduced to encourage players to pick them without that much of a severe consequence to their TC reserves. At the same time, the Sports Bundle has finally been separated into 4 objects. Additionaly, traitors are supposed to be durable and strong, essentialy one-man armies in order to take on the crew. No-slip shoes prevent the usage of water sprays or soaps from easily throwing a traitor's entire round. Due to nuclear operatives being available on lowpop, it was counter-intuitive to pit a single obvious antag against the rest of the crew, so the chameleon kit had its player count restriction removed in order to allow nuclear operatives to be much stealthier, especially the lone ones.

### CODE REVIEW

**Comment by ghost** [comment]
you did it right

**Review by GuyonBroadway — APPROVED** [review]
code looks good

**Comment by DinduStuffin** [comment]
+1
I like that it gives more control over the sports bundles. I think it's ridiculous that you pay money for it and can get one that may or may not be more useful than the others, or just flat out useless in some regards. 

Gimmicks should always be encouraged. I like what this PR is doing. Makes me actually wanna play traitor now.

**Comment by KayeArray** [comment]
tony will defend the sports bundle to his dying breath

**Comment by GuyonBroadway** [comment]
It was an amusing idea in theory but the 20tc lottery was bad cus the things you could get were not woth 20tc.

**Comment by KayeArray** [comment]
yes i am aware it was bad

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -176,3 +155,81 @@
...
+	
+/datum/uplink_item/device_tools/autosurgeon
+	name = "Autosurgeon"
+	desc = "A surgery device that instantly implants you with whatever implant has been inserted in it. Infinite uses. Use a screwdriver to remove an implant from it."
+	item = /obj/item/device/autosurgeon
+	cost = 4
+	surplus = 60
+	
+/datum/uplink_item/implants/microbomb
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/traitor)
```
Comment (ghost) [inline hippiestation/code/modules/uplink/uplink_item.dm:204]: No change log for this

File: hippiestation/code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -176,3 +155,81 @@
...
+
+/datum/uplink_item/implants/macrobomb
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/traitor)
```
Comment (ghost) [inline hippiestation/code/modules/uplink/uplink_item.dm:207]: No change log for this

**Comment by ghost** [comment]
I'm fine with the changes, just update the changelog.
+1

**Comment by GuyonBroadway** [comment]
currently sitting at 1 and 1/3.

**Comment by PopNotes** [comment]
+1 from a nobody

**Comment by ghost** [comment]
capture

**Pull request merged**